### DISCUSSION TITLE
DataGrid defaultPersistanceStorage with 6.8

### DISCUSSION THREAD

**Discussion by RaptorTrap** [serenity-is/Serenity#6939] [Help]
I am currently testing the new 6.8 version and there are few changes regarding SettingStorage.
Before you were able to set this line
`DataGrid.defaultPersistanceStorage = new UserPreferenceStorage();`
in your ScriptInit.ts to set all grids to use the db as the preference storage, this is not working anymore with 6.8:

![image](https://github.com/serenity-is/Serenity/assets/49608875/d90f0400-302e-4f37-8f96-774892945a2e)
How can you do this in 6.8?

**Comment by VictorTomaili** [comment]
Update the serenity extensions package. Check the [changelog](https://github.com/serenity-is/Serenity/blob/master/CHANGELOG.md).

**Reply by RaptorTrap** [reply]
Every Serenity Package is up to date (6.8) and I did check the changelog.
I tested it again and it seems to be working fine but the error is still shown.
I guess its an error in the typings definition?

**Reply by VictorTomaili** [reply]
Typings are up-to-date. You can try to clean the node-modules folder and restore & build the project again.

**Reply by vooba** [reply]
I think the problem is, that https://www.npmjs.com/package/@serenity-is/corelib is still at version 6.7.6

see: https://github.com/serenity-is/Serenity/blob/88ed33f983ec1609f5bbf010dc8df418c46a13ce/packages/corelib/package.json#L3
